.github/workflows/main.yml

```
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+  pull_request:
+
+jobs:
+  deploy_ipfs:
+    runs-on: ubuntu-latest
+    container: node:14
+    env:
+      REACT_APP_SUPPORTED_ETHEREUM_NETWORKS: 'mumbai,kovan,mainnet,polygon,fuji,avalanche'
```

```suggestion
      REACT_APP_SUPPORTED_ETHEREUM_NETWORKS: 'kovan,mumbai,fuji,mainnet,polygon,avalanche'
```
let put first testnet then mainnet